lib/test/tasks/run_eslint.rb

```diff
@@ -3,7 +3,7 @@ class Test::Tasks::RunEslint < Pallets::Task
 
   def run
     execute_system_command(<<~COMMAND)
-      ./node_modules/.bin/eslint --max-warnings 0 --ext .js,.ts,.vue app/javascript/
```

Comment:
(These extension restrictions have been moved (in a different format) to `eslint.config.mjs`.)